bin/main.ts

```diff
@@ -28,4 +30,22 @@ new BlueprintConstruct(app,
     vpc: vpcStack.vpc
   },
   props
-);
\ No newline at end of file
+);
+
+// Added 2nd blueprint for testing custom AMI
+const clusterName = 'customAmi';
+const userData = ec2.UserData.forLinux();
+userData.addCommands(`/etc/eks/bootstrap.sh ${clusterName}`);
+ssp.EksBlueprint.builder()
+    .account(process.env.CDK_DEFAULT_ACCOUNT)
+    .region(process.env.CDK_DEFAULT_REGION)
+    .clusterProvider(new ssp.MngClusterProvider({
+      version: KubernetesVersion.V1_20,
+      customAmi: {
+        machineImage: ec2.MachineImage.genericLinux({'ap-southeast-2': 'ami-0be34337b485b2609'}),
```

Comment:
should the region of the cluster be set tp `ap-southest-2`? CI will operate in us-west-2.